cmd/lncli/commands.go

```diff
@@ -1481,7 +1481,7 @@ func capturePassword(instruction string, optional bool,
 			continue
 		}
 
-		fmt.Println("Confirm password:")
+		fmt.Printf("Confirm password: ")
```

Comment:
Did you test this?

Reply:
Yes, I've compiled it and created a wallet.